### ISSUE TITLE
Issue with Thermostat Setpoint (value is not updated in openhab)

### ISSUE THREAD

**Issue by Togeole** [revenz/openhab#11]
Hi Revenz, 

I have a issue to setup a thermostat.

All is ok under Habdroid and the display is also OK on imperihome.
However, when I change the setpoint value in imperihome, it changes on the imperihome display but it does not change the value on the openhab items resulting that the value is resetted to its original value the next time the imperihome page is refreshed (in my case every 5 seconds).

Items syntax I use:
Number	Thermostat_Temperature	"Thermostat principal [%.2f °C]"	<p_temperature>	(HeatingChart)		{ zwave="38:command=sensor_multilevel,sensor_type=1,refresh_interval=900", imperihab="room:Séjour,label:Température,type:DevTemperature,watts:Thermostat_Temperature" }
Number	Thermostat_Setpoint	"Thermostat principal [%.1f °C]"	<p_thermostat>	(HeatingChart)		{ zwave="38:command=thermostat_setpoint,setpoint_type=1,setpoint_scale=0,refresh_interval=900", imperihab="room:Séjour,label:Thermostat,curmodeid:Thermostat_CurMode,currentTempId:Thermostat_Temperature,minVal:18,maxVal:28,availableModes:Auto-Éteint-Éco-Confort" }
String	Thermostat_CurMode

Can you tell me if I'm doing something wrong?

Thanks!

**Comment by revenz** [comment]
the way I'm using it is.

I have a "virtual" number item for the setpoint
I have a "virtual" average house hold temp (which just updates every x mins or whenever one temperature device receives an update)

I have  a rule that listens to that item and then calls my thermostat and sets the temperature there.     my thermostat I call via a maker task on ifttt.com (my heatpump doesnt have a public API, but does have a IFTTT channel).

**Comment by Togeole** [comment]
Ok, thanks for your answer. 

In my case, I just set the Setpoint on my Thermostat through Openhab or locally and the thermostat starts the boiler actuator if needed. So, I don't have any rules on the Thermostat. Note that other valves on my radiators also start the actuator and there, it's made via a set of rules.

My issue here (and I have the issue either for the Thermostat and the Valves) is that the value that I set via Imperihome only changes the value in Openhab for a few minutes, then the value in Openhab is automatically resetted to the value I had before changing it in Imperihome.

Perhaps it comes from my Z-wave config. I will further check this and I will open a new issue if needed.
